pages/about.html

```diff
@@ -4,9 +4,81 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link rel="stylesheet" href="/styles/about.css">
+  <link rel="stylesheet" href="/styles/index.css">
+  <title>About Me</title>
 </head>
 <body>
+  <nav>
+     <div class="logo">
+      <!-- <img src="/assets/logo.png" alt="logo-masi"> -->
+    </div> 
+    <ul> 
+      <li>
+        <a href="index.html"><em>00.Home</em></a>
+      </li>
+      <li>
+        <a href="about.html"><em>01.About</em></a>
+      </li>
+      <li>
+        <a href="portfolio.html"><em>02.Projects</em></a>
+      </li>
+      <li>
+        <a href="contact.html"><em>03.Contact</em></a>
+      </li>
+    </ul>
+  </nav>
+  <div class="contact-container">
```

Comment:
There are some [semantic HTML tags](https://www.freecodecamp.org/news/semantic-html5-elements/) that could help improve readability.  For example, I'd recommend using the `<aside>` tag here (the aside tag is for a sidebar, or content that's related but not directly part of the main content).
```suggestion
  <aside class="contact-container">
```